pandas/core/frame.py

```diff
@@ -4945,11 +4947,12 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
         Returns
         -------
         quantiles : Series or DataFrame
-            If ``q`` is an array, a DataFrame will be returned where the
-            index is ``q``, the columns are the columns of self, and the
-            values are the quantiles.
-            If ``q`` is a float, a Series will be returned where the
-            index is the columns of self and the values are the quantiles.
+
+            - If ``q`` is an array, a DataFrame will be returned where the
```

Comment:
ok I fixed, but that seems a confusing rule. better to just put blanks around lists in doc-strings always, no?  (checking if taking these out still has warnings). I was trying things so didn't always take out the added whitespace.